network/src/counters.rs

```diff
@@ -33,7 +34,7 @@ pub static LIBRA_CONNECTIONS: Lazy<IntGaugeVec> = Lazy::new(|| {
     register_int_gauge_vec!(
         "libra_connections",
         "Number of current connections and their direction",
-        &["role_type", "network_id", "peer_id", "direction"]
+        &["role_type", "network_id", "self_peer_id", "direction"]
```

Comment:
Why rename this?

Reply:
i'm finding it confusing distinguishing b/w `peer_id` where it's your own id and `peer_id` where it's actually a remote peer id. i was also briefly considering adding a `remote_peer_id` label, but removed it for obvious reasons (label explosion)

Reply:
I've been tackling this in the logging, by leaving the `peer_id` from `network_context` named `peer_id` and naming the external ones `remote_peer_id`.  Seemed a little straightforward to me not to worry about `self` for itself.

Reply:
ok, seems good to me. just gotta be consistent with our convention

Reply:
No skin in the game, but we ought to be consistent across logging and metrics. Perhaps we can use a common const or enum value?